index.bs

```diff
@@ -195,6 +195,18 @@ the user-agent should retain the previously computed layout state if possible.
   values are added on top of any existing containment values.
 </div>
 
+Restrictions and Clarifications {#restrictions}
+===============================
+
+* In situations where [=layout containment=] has no effect (e.g. the
+    element does not generate a principal box), 'subtree-visibility' values also
+    have no effect.
+
+* <a href="https://html.spec.whatwg.org/multipage/rendering.html#replaced-elements">
+    Replaced elements</a>, such as images, do not paint their contents if they
```

Comment:
Suggest further specifying that "do not paint their contents" means not running the parts of step 7.1 or step 7.4 from this algorithm that refer to replaced contents:

https://www.w3.org/TR/CSS21/zindex.html

Reply:
Done. Let me know if this reads ok.